mediacapture-streams/GUM-deny.html

```diff
@@ -2,37 +2,36 @@
 <html>
 <head>
   <title>getUserMedia() triggers error callback when auth is denied</title>
-<link rel="author" title="Dominique Hazael-Massieux" href="mailto:dom@w3.org"/>
-<link rel="help" href="http://w3c.github.io/mediacapture-main/getusermedia.html#error-names">
-<link rel="help" href="http://w3c.github.io/mediacapture-main/getusermedia.html#idl-def-MediaStreamError">
+  <link rel="author" title="Dr. A. Gouaillard" href="mailto:agouaillard@gmail.com"/>
+  <link rel="help" href="http://w3c.github.io/mediacapture-main/getusermedia.html#methods-5">
+  <link rel="help" href="http://w3c.github.io/mediacapture-main/getusermedia.html#navigatorusermediaerrorcallback">
 </head>
 <body>
-<p class="instructions">When prompted, <strong>please deny</strong> access to
```

Comment:
the whitespace-only changes mixed with more substantive changes makes it harder to review, fwiw